docs/library/zephyr.DiskAccess.rst

```diff
@@ -0,0 +1,29 @@
+.. currentmodule:: zephyr
+.. _zephyr.DiskAccess:
+
+class DiskAccess -- access to disk storage
+===============================================
+
+Uses zephyr disk access API.
```

Comment:
would be good to provide a little bit of info about how disk access differs to flash map (eg what happens at the lower level, when to use one or the other)